connect/src/main/java/com/stripe/android/connect/appearance/Typography.kt

```diff
@@ -2,13 +2,20 @@ package com.stripe.android.connect.appearance
 
 import android.os.Parcelable
 import com.stripe.android.connect.PrivateBetaConnectSDK
+import com.stripe.android.connect.appearance.fonts.CustomFontSource
 import dev.drewhamilton.poko.Poko
 import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
 @Parcelize
 @Poko
 class Typography(
+    /**
+     * The font family used throughout the application. Refers to the [CustomFontSource] field.
+     * If null the default will be used.
+     */
+    val fontFamily: String? = null,
```

Comment:
Is there a use case where a consumer would want to read this value? I feel like I would expect this to never be `null` even if it was the default. Maybe either name it `fontFamilyOverride` or make it non-null and have it default to the actual font?

Reply:
The problem is that what default font is used is contained within the JS SDK and thus opaque to us. Setting null is what instructs the JS to use the default.

I like `fontFamilyOverride`, but one of the goals here is to match as much as possible the iOS and JS SDK's which use the `fontFamily` name so I'm going to keep it as-is